src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -6158,11 +6180,11 @@ quantitykind:IonConcentration
   a qudt:QuantityKind ;
   dcterms:description "\"Ion Concentration\" is the number of ions per unit volume. Also known as ion density."^^qudt:LatexString ;
   qudt:exactMatch quantitykind:IonDensity ;
-  qudt:hasDimensionVector qkdv:A0E0L-3I0M0H0T0D0 ;
+  qudt:hasDimensionVector qkdv:A1E0L-3I0M0H0T0D0 ;
```

Comment:
This change opens up a larger discussion: Is a numeric count dimensionally the same as "amount of material" measured in moles? 

Argument in favor: Yes, you can just choose which unit to use, be it a mole or a number
Argument against: No, a number or count is a dimensionless concept, not the SI concept of "amount of material" that is dimensionally encoded as A1 in our dimension vector.

In the past I have considered number as a dimensionless concept.

Thoughts?

Reply:
Looking through the Web, I think the term 'ion concentration' is not sufficiently defined. It refers to the amount of a substance in a solvent (in chemistry), and might either be SI Amount (mol) or a relative count (e.g. ppm, which is mg/L), and there is Molality (did not delve into that) and Normality (neither). 

Then there is ion density (exact match in qudt), which does not seem to be an exact match at all: it is either number density (#/m3), mass density(kg/m3) or charge density(coulomb/m3)

So for our system, in which a quantitykind is associated with one dimension vector, we need quite a bunch of them to cover ion concentration and ion density.

I touched it because IFC thinks IonConcentration is g/L, which until now I thought was wrong, but now (thanks to your watchful eye) I believe it is actually correct, and I think I need to add at least that special kind.

If we wanted to be more correct than we are currently, we may consider
 
IonAmountConcentration
IonCountConcentration
IonMassConcentration (that's the one that IFC thinks is relevant)
[omit the more funky ones with Normality and Molality]

IonNumberDensity
IonMassDensity
IonChargeDensity

Reply:
I think you are on the right track. We only need to distinguish the quantity kinds that have distinct dimension vectors. Others, like molarity and molality, have to do with moles per volume of solute or moles per volume of final solution, so they have the same dimensionality. We can handle those cases with rdfs:seeAlso. 

Look at our description for quantitykind:Concentration 

So we should list the possibilities by their quantity kinds, as in:
- count per volume
- moles per volume (aka Concentration, AmountOfSubstancePerVolume, AmountOfSubstanceConcentration)
- mass per volume. (aka Density or MassDensity)
- charge per volume

Have to run...

Reply:
right now, all I need is a quntitykind for IonConcentration that uses amount of substance per volume. I'll add just that as `qk:AmountOfSubstanceIonConcentration`

Reply:
done